rust/worker/src/blockstore/arrow/blockfile.rs

```diff
@@ -275,6 +276,30 @@ impl<'me, K: ArrowReadableKey<'me> + Into<KeyWrapper>, V: ArrowReadableValue<'me
         None
     }
 
+    pub(super) async fn load_blocks(&self, block_ids: Vec<Uuid>) -> () {
+        let mut futures = Vec::new();
+        for block_id in block_ids {
+            futures.push(self.get_block(block_id));
```

Comment:
I think something about this feels a bit off, since if its cached we dispatch it needlessly